bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/messages.properties

```diff
@@ -711,6 +711,9 @@ WorkbenchPreference_workbenchSaveIntervalError=The workbench save interval shoul
 WorkbenchEditorsAction_label=S&witch to Editor...
 WorkbookEditorsAction_label=&Quick Switch Editor
 
+WorkbenchPreference_largeViewLimit=Show not more than given number of children per parent in views:
```

Comment:
I'm not sure about the wording here, seems lengthy despite using parent/child terminology.

Maybe just "Limit displayed children per parent in tree views"?

Reply:
> Maybe just "Limit displayed children per parent in views"?

OK, see alternatives

- "Show not more than given number of children per parent in views: 1000"
- "Limit displayed children per parent in views: 1000"
- "Limit displayed elements in views with increment: 1000"

@trancexpress : what about last one?

Reply:
We've discussed that once again, proposal would be "Default increment for displaying elements in UI: 1000".
This doesn't mean it is supposed to be in every view, and doesn't mention parent/child but just "increment".